src/components/site/BreadCrumbLinkItem.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const BreadcrumbLinkItem = props => {
+  const { link, breadcrumbText, title } = props;
+  return (
+    <li>
```

Comment:
can we give this a class of `dg_breadcrumb`?

Reply:
I had some questions regarding this item. Would the class dg_breadcrumb inherit the styles of .dg_breadcrumbs li? For the sake of reducing the amount of markup, I'm thinking through the benefits of including dg_breadcrumb on the list item vs not.

Reply:
As requested, updated the li within BreadCrumbLinkItem to include a class dg_breadcrumb. Also updated the stylesheet to reflect this.